orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/support/CustomTriggerDeserializerSupplier.kt

```diff
@@ -29,5 +30,5 @@ import com.netflix.spinnaker.orca.api.pipeline.models.Trigger
  */
 interface CustomTriggerDeserializerSupplier {
   val predicate: (node: JsonNode) -> Boolean
-  val deserializer: (node: JsonNode) -> Trigger
+  val deserializer: (node: JsonNode, parser: JsonParser) -> Trigger
```

Comment:
Update the javadoc to mention JsonParser?

Reply:
How the javadoc is updated? I am not sure how to do that.